networkx/algorithms/dag.py

```diff
@@ -139,6 +141,14 @@ def topological_sort(G, nbunch=None, reverse=False):
 
     if nbunch is None:
         nbunch = G.nodes_iter()
+        def direct_descendants(w):
+            return G[w]
+    else:
+        nbunch_index = {x: i
+                        for i, x in enumerate(nbunch)}
```

Comment:
Considering you have one `for`, it would be best to stick to one line.
Line breaks are advised when dealing with more than one `for`, as far as I understand.